src/components/candidates/CandidatesList.vue

```diff
@@ -1,286 +1,27 @@
-    linkedinUrl: "linkedinUrl 24",
-    feedback: "feedback 24",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/110.jpg",
-    id: "24",
-  },
-  {
-    position: "position 25",
-    username: "username 25",
-    linkedinUrl: "linkedinUrl 25",
-    feedback: "feedback 25",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/1174.jpg",
-    id: "25",
-  },
-  {
-    position: "position 26",
-    username: "username 26",
-    linkedinUrl: "linkedinUrl 26",
-    feedback: "feedback 26",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/411.jpg",
-    id: "26",
-  },
-  {
-    position: "position 27",
-    username: "username 27",
-    linkedinUrl: "linkedinUrl 27",
-    feedback: "feedback 27",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/855.jpg",
-    id: "27",
-  },
-  {
-    position: "position 28",
-    username: "username 28",
-    linkedinUrl: "linkedinUrl 28",
-    feedback: "feedback 28",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/1213.jpg",
-    id: "28",
-  },
-  {
-    position: "position 29",
-    username: "username 29",
-    linkedinUrl: "linkedinUrl 29",
-    feedback: "feedback 29",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/921.jpg",
-    id: "29",
-  },
-  {
-    position: "position 30",
-    username: "username 30",
-    linkedinUrl: "linkedinUrl 30",
-    feedback: "feedback 30",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/420.jpg",
-    id: "30",
-  },
-  {
-    position: "position 31",
-    username: "username 31",
-    linkedinUrl: "linkedinUrl 31",
-    feedback: "feedback 31",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/863.jpg",
-    id: "31",
-  },
-  {
-    position: "position 32",
-    username: "username 32",
-    linkedinUrl: "linkedinUrl 32",
-    feedback: "feedback 32",
-    avatarUrl:
-      "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/544.jpg",
-    id: "32",
-  },
-];
+import { useCandidateStore } from "../../stores/candidates";
+import { ref } from "vue";
+
+const { getAllCandidates } = useCandidateStore();
+const candidatesList = ref([]);
+const currentCandidateList = ref([]);
+const candidatePagesNum = ref(0);
+async function getCandidates() {
+  try {
+    const { data } = await getAllCandidates();
+    candidatesList.value = data;
+
+    candidatePagesNum.value = Math.ceil(data.length / 8);
+    currentCandidateList.value = [...data].slice(0, 8);
+  } catch (e) {
+    console.log(e);
+  }
+}
+function changePage(start, end) {
```

Comment:
My opinion better to use server side pagination. 
If we will have more 1000 candidates in future how long this page will be load?

Reply:
i agree, but we have max 100 obj in mockApi, i can change it, if you want

Reply:
We can't lose scalable